src/js/fields/FileField.jsx

```diff
@@ -122,10 +122,10 @@ export default class FileField extends React.Component {
               });
               const itemSchema = schema.items[index];
               const attachmentIdSchema = {
-                $id: `${idSchema.$id}_${index}_atachmentId`
+                $id: `${idSchema.$id}_${index}_attachmentId`
               };
               const attachmentNameSchema = {
-                $id: `${idSchema.$id}_${index}_atachmentName`
+                $id: `${idSchema.$id}_${index}_attachmentName`
```

Comment:
These two things are what I was _actually_ changing here.

Reply:
This has been misspelled for a while so changing the name could be considered a breaking change. I don't think that's a problem in reality though.